api/routes/notes/get-note.routes.js

```diff
@@ -1,10 +1,13 @@
 const { StatusCodes } = require('http-status-codes');
 const canReadNote = require('../../middlewares/auth/can-read-note.middleware');
+const canWriteNote = require('../../middlewares/auth/can-write-note.middleware');
+const canCreateNote = require('../../middlewares/auth/can-create-note.middleware');
+const isAuthor = require('../../middlewares/auth/is-author.middleware');
 const supabase = require('../../db');
 
 const router = require('express').Router();
 
-router.get('/:noteId', canReadNote, async (req, res) => {
+router.get('/users/:username/notes/:noteId', isAuthor, async (req, res) => {
```

Comment:
Not sure the `isAuthor` middleware is adapted here: a user which has read access to a note could want to get specific data about that note. 
Could we use the `canReadNote` middleware instead? Or is it really important that we use the `isAuthor` one?